documentation/library_guide/parallel_api/iterators.rst

```diff
@@ -92,7 +92,8 @@ header.  All iterators are implemented in the ``oneapi::dpl`` namespace.
   a functor whose index operator defines the mapping from the ``permutation_iterator`` index to the index of the
   source iterator. The ``permutation_iterator`` is useful in implementing applications where noncontiguous
   elements of data represented by an iterator need to be processed by an algorithm as though they were contiguous.
-  An example is copying every other element to an output iterator.
+  An example is copying every other element to an output iterator. The source iterator cannot be a host-side iterator
```

Comment:
My suggestion:

An example is copying every other element to an output iterator. The source iterator cannot be a host-side iterator when algorithms are executed with device policies.

Reply:
resolved.
@dcbenito  could you please re-review the changes?

Reply:
My suggestion: The source iterator cannot be a host-side iterator in cases where algorithms are executed with device policies.